src/code_translation/NodeSelector.py

```diff
@@ -99,13 +118,14 @@ def visit_Assign(self, node: cst.Assign):
     def resolve_Call(self, node: cst.Call) -> Optional[Node]:
         args = list(self.parse_NonKwArg(arg) for arg in node.args if not arg.keyword)
         kwargs = dict(self.parse_KwArg(arg) for arg in node.args if arg.keyword)
-
         result = None
 
         if isinstance(node.func, cst.Name):  # calls to directly imported library function. e.g. read_sql
             func_alias = node.func.value
             if func_alias not in self.library_methods:
-                return
```

Comment:
This causes function calls to methods that do not have a inputModule to be None, which is kinda shitty for everything outside the pandas/dask ecosystem

Reply:
I think in this case it is most appropriate to throw an UnresovableCSTNode exception. This will make this resolve to the CSTNode itself. Don't know if i forgot that in my exceptions-branch or if there was something that speaks against this